plugin/pki/path_venafi_cert_enroll.go

```diff
@@ -270,7 +270,7 @@ func (b *backend) pathVenafiCertObtain(ctx context.Context, req *logical.Request
 	if !role.NoStore {
 		if role.StoreBy == storeByCNString {
 			//Writing certificate to the storage with CN
-			b.Logger().Debug("Putting certificate to the certs/" + reqData.commonName)
+			b.Logger().Debug(fmt.Sprintf("Putting certificate to the certs/" + reqData.commonName))
```

Comment:
also using fmt.Sprintf() seems to change the syntax of this expression: shouldn't it be:
`b.Logger().Debug(fmt.Sprintf("Writing certificate to the certs/%s",reqData.commonName))`

Reply:
done

Reply:
removed the fmt.Sprintf method given that the variable string can be concatenated and is at the end of the message